drafts/draft-ietf-ippm-ioam-data.xml

```diff
@@ -2010,10 +1926,11 @@ IOAM proof of transit option of IOAM POT Type 0:
       <t>The data elements of IOAM can be used for network reconnaissance,
       allowing attackers to collect information about network paths,
       performance, queue states, buffer occupancy and other information. Note
-      that in case IOAM is used in "immediate export" mode, the IOAM related
-      trace information would not be available in the customer data packets,
-      but would be exported by every IOAM node. IOAM data export and securing
-      IOAM data export is outside the scope of this document.</t>
+      that in case IOAM is used in "immediate export" mode (TBD - to be 
+	  updated), the IOAM related trace information would not be available in 
```

Comment:
Could we be specific about what "information" would not be available. E.g.
s/information/information (i.e. node data[i])/